pneumonia-detection/build.gradle

```diff
@@ -12,11 +12,11 @@ repositories {
 }
 
 dependencies {
-    implementation platform("ai.djl:bom:0.12.0")
+    implementation platform("ai.djl:bom:0.20.0")
     implementation "ai.djl:api"
 
     runtimeOnly "ai.djl.tensorflow:tensorflow-engine"
-    runtimeOnly "ai.djl.tensorflow:tensorflow-native-auto"
+    runtimeOnly "ai.djl.tensorflow:tensorflow-native-cpu"
```

Comment:
```suggestion
```

Reply:
This is tensorflow-engine dependency, should we keep the native?

Reply:
We don't have any -auto package. only engine jar is needed

Reply:
No, we don't. all native package need a classifier

Reply:
OK